consultation_analyser/consultations/download_consultation.py

```diff
@@ -25,14 +26,16 @@ def consultation_to_json(consultation):
     attrs = {}
 
     themes = []
-    theme_records = Theme.objects.filter(answer__question__section__consultation=consultation)
-    for theme in theme_records:
-        theme_attrs = select_keys_from_model(
-            theme, ["topic_id", "topic_keywords", "summary", "short_description"]
-        )
+    latest_processing_run = consultation.latest_processing_run
+    if latest_processing_run:
+        theme_records = Theme.objects.filter(processing_run=latest_processing_run)
```

Comment:
Shall we create `themes()` (or `themes`?) on `ProcessingRun`? Useful here and in `llm_summariser` and inside `ProcessingRun` itself

Reply:
Now updated.